WalletWasabi.Tests/UnitTests/AbandonedTasksTests.cs

```diff
@@ -61,7 +61,7 @@ static async Task BusyWorkerTask(CancellationTokenSource cts)
 				try
 				{
 					// Wait for the cancellation.
-					await Task.Delay(-1, cts.Token).ConfigureAwait(false);
+					await Task.Delay(-1, cts.Token);
```

Comment:
This is a local submethod, there should be ConfigureAwait(false).

Reply:
Pushed changes.